README.md

```diff
@@ -2,7 +2,7 @@
 
 # hyperimport
 
-⚡ TypeScript imports on steriods. Import c, rust, zig etc. files in your typescript code and more.
+⚡ TypeScript imports on steroids. Import C, Rust, Zig, etc in your TypeScript code and more.
```

Comment:
Thanks for pointing out the first typo. As for the next changes, I would recommend to keep them unchanged because that line has been used everywhere and changing that would make it different.

Reply:
Awesome, I reverted to the original format but it seems like the name should be still in capital case, right?

Reply:
Agree.